users/api/metrics.go

```diff
@@ -120,11 +120,20 @@ func (ms *metricsMiddleware) SendPasswordReset(ctx context.Context, host, email,
 	return ms.svc.SendPasswordReset(ctx, host, email, token)
 }
 
-func (ms *metricsMiddleware) ListMembers(ctx context.Context, token, groupID string, offset, limit uint64, gm users.Metadata) (users.UserPage, error) {
+func (ms *metricsMiddleware) ListMembers(ctx context.Context, token, groupID string, state string, offset, limit uint64, gm users.Metadata) (users.UserPage, error) {
 	defer func(begin time.Time) {
 		ms.counter.With("method", "list_members").Add(1)
 		ms.latency.With("method", "list_members").Observe(time.Since(begin).Seconds())
 	}(time.Now())
 
-	return ms.svc.ListMembers(ctx, token, groupID, offset, limit, gm)
+	return ms.svc.ListMembers(ctx, token, groupID, state, offset, limit, gm)
+}
+
+func (ms *metricsMiddleware) ChangeUserStatus(ctx context.Context, token string, id string) (err error) {
+	defer func(begin time.Time) {
+		ms.counter.With("method", "remove_user").Add(1)
```

Comment:
@0x6f736f646f Since we will have only these two states, and it's unlikely that will change in foreseeable future, the @manuio proposal sounds OK. It's a cleaner service interface and we can utilize the same repository implementation.